### ISSUE TITLE
Is it possible to generate validation error????

### ISSUE THREAD

**Issue by siddiquemu** [facebookresearch/Detectron#616]
![trainloss](https://user-images.githubusercontent.com/42333767/44014651-a7b9f952-9e92-11e8-816a-0fcf1a4b1222.png)
![train_accuracy](https://user-images.githubusercontent.com/42333767/44014652-a7c57ef8-9e92-11e8-8e57-41a39f8bd594.png)

I can generate training loss and accuracy vs iteration to observe the training progress for my own dataset. I am curious to check the validation loss and accuracy also to stop the over fitting. can anyone help me to get validation error during training?

Network use both training and validation set for training. There is no validation performance, right?

**Comment by gadcam** [comment]
@siddiquemu Looks like this should be a partial answer to your question https://github.com/facebookresearch/Detectron/blob/b3c93df2cecca1139f73d005b9dfcd83ef55c16d/FAQ.md#q-how-do-i-compute-validation-ap-during-training

**Comment by ccpocker** [comment]
@siddiquemu Could you share your code for visualization train loss and train accurancy

**Comment by ccpocker** [comment]
@siddiquemu  Very appreciate for your sharing

**Comment by RakhithaRR** [comment]
How did you manage to set up a custom database other than COCO. I'm also trying to do the same thing, but with no luck. Any help is appreciated.

**Comment by siddiquemu** [comment]
@RakhithaRR  You can just prepare training instances similar to .json files of coco dataset.

**Comment by Zhaosoi** [comment]
@siddiquemu Hi, can you get validation loss and accuracy curve during training now? Could you share your code. Many thanks!
